README.md

```diff
@@ -1,80 +1,19 @@
-# Smart-Contracts for [The Sandbox](https://www.sandbox.game)
+# TSB contracts
 
-[www.sandbox.game](https://www.sandbox.game)
+This workspace uses [Nx, a Smart, fast and extensible build system.](https://nx.dev) 
 
-The Sandbox is a user-generated content (UGC) gaming platform that will empower creators through digital ownership and monetization of 3D voxel creations made and shared by users around the world.
+## Understand this workspace
 
-Learn more:
+Run `nx graph` to see a diagram of the dependencies of the projects.
 
-- Website: www.sandbox.game
-- Discord: https://discordapp.com/invite/vAe4zvY
-- Telegram: https://t.me/sandboxgame
-- Medium: https://medium.com/sandbox-game
+Optionally every package can implement the following targets that will be run on the CI:
 
-# Blockchain Features
+- test
+- coverage: run coverate, CI expect to have at least 80% coverage on packages.
+- lint
+- format: format the code with prettier.
+- deploy: to test hardhat deployments.
 
```

Comment:
Could we have a line or two telling users where they should create their new packages, as it isn't obvious what the immediate steps are?